src/StoryContract.sol

```diff
@@ -19,29 +19,67 @@ pragma solidity 0.8.17;
 ///////////////////// IMPORTS /////////////////////
 
 import { ERC165 } from "openzeppelin/utils/introspection/ERC165.sol";
-import { IStory, NotTokenCreator, NotTokenOwner, StoryNotEnabled, TokenDoesNotExist } from "./IStory.sol";
+import { IStory } from "./IStory.sol";
+
+/*//////////////////////////////////////////////////////////////////////////
+                            Custom Errors
+//////////////////////////////////////////////////////////////////////////*/
+
+/// @dev story additions are not enabled
+error StoryNotEnabled();
+
+/// @dev token does not exist
+error TokenDoesNotExist();
+
+/// @dev caller is not the token owner
+error NotTokenOwner();
+
+/// @dev caller is not the token creator
+error NotTokenCreator();
+
+/// @dev caller is not a story admin
+error NotStoryAdmin();
+
+/*//////////////////////////////////////////////////////////////////////////
+                            Story Contract
+//////////////////////////////////////////////////////////////////////////*/
 
 abstract contract StoryContract is IStory, ERC165 {
 
-    ///////////////////// STORAGE VARIABLES /////////////////////
+    /*//////////////////////////////////////////////////////////////////////////
+                                State Variables
+    //////////////////////////////////////////////////////////////////////////*/
 
     bool public storyEnabled;
 
-    ///////////////////// MODIFIERS /////////////////////
+    /*//////////////////////////////////////////////////////////////////////////
+                                Modifiers
+    //////////////////////////////////////////////////////////////////////////*/
 
     modifier storyMustBeEnabled {
         if (!storyEnabled) { revert StoryNotEnabled(); }
         _;
     }
 
-    ///////////////////// CONSTRUCTOR /////////////////////
+    /*//////////////////////////////////////////////////////////////////////////
+                                Constructor
+    //////////////////////////////////////////////////////////////////////////*/
 
     /// @param enabled is a bool to enable or disable Story addition. This cannot be undone later.
     constructor(bool enabled) {
         storyEnabled = enabled;
     }
 
-    ///////////////////// STORY FUNCTIONS /////////////////////
+    /*//////////////////////////////////////////////////////////////////////////
+                                Story Functions
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @dev function to set story enabled/disabled
+    /// @dev requires story admin
+    function setStoryEnabled(bool enabled) external {
+        if (!_isStoryAdmin(msg.sender)) { revert NotStoryAdmin(); }
```

Comment:
not sure if it cuts down on bytecode size but can technically remove `{` and `}`

Reply:
that's true and looks cleaner